datasets/text2log/README.md

```diff
@@ -0,0 +1,147 @@
+---
+annotations_creators:
+- machine-generated
+language_creators:
+- machine-generated
+languages:
+- en-US
+licenses:
+- unknown
+multilinguality:
+- monolingual
+pretty_name: 'text2log'
+size_categories:
+- unknown
+source_datasets:
+- original
+task_categories:
+- conditional-text-generation
+task_ids:
+- machine-translation
+---
+
+# Dataset Card for text2log
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** none
+- **Repository:** https://github.com/alevkov/text2log
+- **Paper:** https://github.com/alevkov/text2log
+- **Leaderboard:** none
+- **Point of Contact:** https://github.com/alevkov
+
+### Dataset Summary
+
+data set containing 100,000 simple English sentences selected and filtered from enTenTen15 and their translation into First Order Logic (FOL) using ccg2lambda
+
+### Supported Tasks and Leaderboards
+
+'semantic-parsing': The data set is used to train models which can generate FOL statements from natural language text
+
+### Languages
+
+en-US
+
+## Dataset Structure
+
+### Data Instances
+
+{
+'clean':'All things that are new are good.',
+'trans':'all x1.(_thing(x1) -> (_new(x1) -> _good(x1)))'
+}
```

Comment:
````suggestion
```
{
'clean':'All things that are new are good.',
'trans':'all x1.(_thing(x1) -> (_new(x1) -> _good(x1)))'
}
```
````